pipeline/cbt_librbd.conf

```diff
@@ -1,7 +1,7 @@
 input {
 	file
 	{
-	 	path => [ "${LOGSTASH_DIR}/**/results/**/**/json_output.*" ]
+	 	path => [ "${LOGSTASH_DIR}/librbd/**/results/**/**/json_output.*" ]
```

Comment:
that librbd prefix is extra to cbd results, please add to the doc that... That someone need to create those dirs.